### DISCUSSION TITLE
Showing a help screen

### DISCUSSION THREAD

**Discussion by jeeger** [kovidgoyal/kitty#4076] [General]
I'm looking for a way to show help about my keybindings. I'd want to display some fixed text in a kitty overlay. However, I can't seem to find a way to do this, except creating a help file and spawning a new pager process for that file.

Is there a simpler way to display some fixed text in a kitty window?

**Comment by kovidgoyal** [comment]
On Wed, Sep 29, 2021 at 12:03:18AM -0700, Jan Seeger wrote:
> I'm looking for a way to show help about my keybindings. I'd want to display some fixed text in a kitty overlay. However, I can't seem to find a way to do this, except creating a help file and spawning a new pager process for that file.
> 
> Is there a simpler way to display some fixed text in a kitty window?

```
map f1 launch --type=overlay less /path/to/help/file
```
